services/dashboard/app/root.tsx

```diff
@@ -201,8 +204,40 @@ function App() {
 				</header>
 
 				<div className="flex-1">
-					<p>ReactFlow Testing</p>
+					<p>Mock Server Workflow Testing</p>
 					{data.workflow && <p> {data.workflow.edges[0].data.sourceLabel} TO {data.workflow.edges[0].data.targetLabel}</p>}
+					
```

Comment:
Amazing! I just take a look and is awesome, just one thing, can you move this to another route? create one in the `app/routes` folder, and show up this component please

Reply:
Sure thing! Just a question, should I move only the line you mentioned or the whole react flow?

Reply:
All the react flow! Github doesnt get multi-line hahaha